app/utils.py

```diff
@@ -42,7 +42,7 @@ def url_with_token(data, url, config, base_url=None):
     return base_url + token
 
 
-def get_template_instance(template, values):
+def get_template_instance(template, values=None):
```

Comment:
Are things okay and able to still flow through if `values` is indeed `None`, or is there error handling further down the call stack that'll handle it?